### DISCUSSION TITLE
Many HTTP status code 524 errors for API responses

### DISCUSSION THREAD

**Discussion by kazuhisa-wada** [perplexityai/api-discussion#61] [General]
Hi,

We're facing many HTTP status code 524 errors of responses for API requests while rate limits are not hit obviously.
Does anyone know if there is additional rate limit other than what is listed here https://docs.perplexity.ai/guides/rate-limits ?

Or is there any good tips to prevent  HTTP 524 errors ?

This situation seems to have started since late last week. Also wanted other guys to join this thread who have been facing this issue.

**Comment by kazuhisa-wada** [comment]
@shubhang98 
Could you kindly point me to where I should raise this ?
I believe this is sort of issue and so as a temporary action I just put it on this thread. 
Here is good place to raise?

**Reply by shubhang98** [reply]
can you email api@perplexity.ai? one of our team members can look into this.

**Reply by kazuhisa-wada** [reply]
ok, thanks for your suggestion.
